src/test/groovy/org/gradle/android/TaskAvoidanceTest.groovy

```diff
@@ -0,0 +1,55 @@
+package org.gradle.android
+
+import org.gradle.util.GradleVersion
+import org.gradle.util.VersionNumber
+import spock.lang.Unroll
+
+class TaskAvoidanceTest extends AbstractTest {
+    @Unroll
+    def "tasks are avoided with #gradleVersion and Android plugin #androidVersion"() {
+        assert gradleVersion instanceof GradleVersion
+        assert androidVersion instanceof VersionNumber
+
+        println "> Using Android plugin $androidVersion"
+        println "> Running with $gradleVersion"
```

Comment:
Use same order in asserts as in printlns

Reply:
Why do we even have these asserts and printlns?

Reply:
Not sure, was copypasta from here: https://github.com/gradle/android-cache-fix-gradle-plugin/blob/master/src/test/groovy/org/gradle/android/CrossVersionOutcomeAndRelocationTest.groovy#L20

The test method name should have the versions in it. Will remove